### DISCUSSION TITLE
Asking to login again

### DISCUSSION THREAD

**Discussion by infocodiste** [Web3Auth/Web3Auth#722] [Help]
I am using TorusWalletConnectorPlugin
![Screenshot 2022-10-13 at 7 43 43 PM](https://user-images.githubusercontent.com/78797181/195621533-aa3d7802-23ed-4f55-aaf0-15dbb2a9c666.png)
on Clicking the "Open Wallet" button/link, a window pops up and shows the following:
![Screenshot 2022-10-13 at 7 47 07 PM](https://user-images.githubusercontent.com/78797181/195622000-6c563af3-fd52-4536-b5ec-20bbf0f39092.png)
It was not asking me to login again till today, I am using this plugin for like more than 10 days and on clicking the "Open Wallet" button, it used to show me my account and all the information related to my account. Then suddenly why it is asking me login in?

**Comment by rinzler99** [comment]
Hey,
This might have happened because your session expired. I would suggest you to  loging  again if the problem persists, let us know. Thanks!

**Comment by thangtranth** [comment]
Do anyone know the issue? 
I am testing the test website:  https://main.dmmzgprvd8gqq.amplifyapp.com
When I click Open Wallet in TorusWalletConnectorPlugin. It forces me to log-in again. 
![image](https://user-images.githubusercontent.com/60476800/223016989-1a2fa507-4938-4ce7-98ef-681899677e87.png)

And when I log in using the same email, it leads to different account address.
Please help.
